src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkDirectional.java

```diff
@@ -103,6 +103,9 @@ public void tick(Block block, SlimefunItem slimefunItem, Config config) {
 
                 @Override
                 public void uniqueTick() {
```

Comment:
Nitpick as it doesn’t effect this fix, but I think the if statement needs to happen after the calculation? Because uniqueticks are meant to happen before the individual ticks?

Reply:
Oh I didn't realize that, will do!